<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select your subject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plusBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplyBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
